src/frame/js/actions/faye.js

```diff
@@ -161,10 +163,11 @@ function updateUser(currentAppUser, nextAppUser, client) {
                 setItem(`${appId}.sessionToken`, nextAppUser.sessionToken);
             }
 
+            dispatch(disconnectFaye());
```

Comment:
would it also work if `disconnectFaye()` is included in the actions array passed to `batchActions(actions)` instead?